plugin/buycourses/src/buy_course_plugin.class.php

```diff
@@ -164,6 +171,131 @@ public function uninstall()
         $this->manageTab(false);
     }
 
+    public function update()
+    {
+        $table = self::TABLE_GLOBAL_CONFIG;
+        $sql = "SHOW COLUMNS FROM $table WHERE Field = 'global_tax_perc'";
+        $res = Database::query($sql);
+
+        if (Database::num_rows($res) === 0) {
+            $sql = "ALTER TABLE $table ADD (
+                sale_email varchar(255) NOT NULL,
+                global_tax_perc int(11) unsigned NOT NULL,
+                tax_applies_to int(11) unsigned NOT NULL,
+                tax_name varchar(255) NOT NULL,
+                seller_name varchar(255) NOT NULL,
+                seller_id varchar(255) NOT NULL,
+                seller_address varchar(255) NOT NULL,
+                seller_email varchar(255) NOT NULL,
+                next_number_invoice int(11) unsigned NOT NULL,
```

Comment:
Por compatibilidad futura con otros sistemas de bases de datos (lo que viene en una de las versiones 2.*), por favor no usar marcadores de alineación de campos para los int.
```
next_number_invoice int(11) unsigned not null,  -- NO
next_number_invoice int unsigned not null,  -- YES
```
See coding standards in the wiki section.

Reply:
Corregido